### PR TITLE
DPWEBCORE-927: Adding performance report

### PR DESCRIPTION
**Pull request by ameliamyfanwy**
@bbc/web-core-devx

# Problem

-   Add performance report to status page
-   Wider context: https://jira.dev.bbc.co.uk/browse/DPWEBCORE-927

# Solution

-   Saved copy of report in public folder which automatically gets copied to build folder and therefore deployed
-   Added link to page

### CODE REVIEW

File: src/app/App.test.tsx [hunk 1]
```diff
@@ -2,8 +2,25 @@ import React from "react";
...
   const { getByText } = render(<App />);
   const headerElement = getByText(/WebCore Capability Run Books/i);
   expect(headerElement).toBeInTheDocument();
 });
+
+test("Renders a link to performance report using the public URL", () => {
+  (process.env as MockEnvironment).PUBLIC_URL = "https://www.google.com/";
+  const { container } = render(<App />);
+  const reportLink = container.querySelector(
+    "a[href='https://www.google.com/webcore-performance-report-august-v2.pdf']"
```
Comment (jameshd) [inline src/app/App.test.tsx:23]: Does querySelector support the contains `*=` operator, we could drop the google.com stuff that way. 

[Answered my own question, yes it does. ](https://drafts.csswg.org/selectors/#attribute-substrings)

```suggestion
    "a[href*='/webcore-performance-report-august-v2.pdf']"
```

Also, might be useful to just put the relative path to the doc so not including the `PUBLIC_URL` bit.

**Comment by jameshd** [comment]
👏  Great stuff Amelia! 👏

**Review by abrarkhan1234 — APPROVED** [review]

File: src/app/App.test.tsx (hunk 1 above)
Comment (benwainwright) [reply src/app/App.test.tsx:23]: If we did that we'd no longer be asserting on the thing we are actually testing, James - which is that the URL is constructed from the value that we assigned to `process.env.PUBLIC_URL`

Comment (benwainwright) [reply src/app/App.test.tsx:23]: > Also, might be useful to just put the relative path to the doc so not including the PUBLIC_URL bit.

This is the approach recommended by the CRA documentation

**Review by jameshd — APPROVED** [review]

**Review by MattLongCode — APPROVED** [review]

**Pull request merged**